docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,197 @@
+
+
+### Improved nodes view on the Inventory page
+
+In our continuing endeavor to enhance user experience, we further improved the **Inventory** page by improving the **Nodes** tab. With this change, you can get more context on your inventory objects, such as node, and more actionable information for exploring and resolving possible issues.
+
+Here is the change as part of PMM 2.38.0:
+
+- You can now see the number of agents running on any particular node. When you click on any node, the UI navigates to the view of agents, which is filtered to display only agents related to that specific node. 
+
+To see the details of the agents running, do the following:
+
+On the **Nodes** tab, under the **Monitoring** column, click **OK** or **Failed** depending on the status of the node that you have selected. A page that provides the user with crucial information regarding the total number of agents deployed on that node is displayed.
+    ![!image](../_images/PMM_Inventory_Node_Agent_Properties.png)
+
+- You can also view comprehensive information about each agent, including key attributes such as node type, IP address, and associated services. This gives you a complete overview of the nodes at a glance.
+
+    ![!image](../_images/PMM_Inventory_Node_Selection.png)
+
+
+- You can check the health status of the agents to see if they are running or have failed.
+
+### Backup enhancements
+
+#### Better folder management for enhanced PMB compatibility 
+We’ve added a new **Folder** field to the Create Backup pages. Use this to set a specific target directory within the selected local or S3-compatible location for the backup.  
+
+Organizing backups in folders makes grouping PBM backups for clusters easier, and it improves PMM-PBM (Percona Backup for MongoDB) integration workflows. 
+
+The **Folder** field is automatically populated with the value of the cluster label. You can change this default folder from PMM’s Advanced Sett`ings, but make sure you understand how your custom folder will impact PBM integration workflows.
+
+
+#### Sharded cluster MongoDB backup support 
+This release introduces support for working with backups for sharded clusters. PMM handles the backup process end-to-end, but restoring such artifacts is currently possible only via the CLI, using Percona Backup for MongoDB.
+
+For information on restoring sharded backups, check the [PBM documentation](https://docs.percona.com/percona-backup-mongodb/usage/restore.html)
+
+
+### SQL comments support in QAN
+
+You can now collect labels from the comments placed in SQL queries. You can tag the query by key value in the query comment. This helps in query management and optimization. This feature is currently supported for MySQL and PostgreSQL.
+
+
+For example:
+
+    ```sh
+    SELECT * /* cluster=’east’ */ FROM city;   
+    ```
+You can enable this feature as follows:
+CLI - Pass the parameter `comments-parsing`
+UI - 
+
```

Comment:
``` suggestion
### New default Alert Template
Percona will provide PMM users with a new default Alert Template for Percona Alerting (Link to Alerting Doc) to use when you need to monitor your Node status, and we will inform you if the Node goes down.  This Alert template can be used with all other Alert Templates and consume the same notification channels. 

Note: This feature is based on PMM Agent to be presented on Database Node and be healthy. Also, this will require PMM Client to be on at least the 2.38 version.  
```
Based on https://jira.percona.com/browse/PMM-9544